### PR TITLE
Deprecate TiDB Binlog

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

This PR adds deprecation warnings or notes to docs related to TiDB Binlog

Note: The status of TiDB Binlog in [TiDB 功能概览 | PingCAP 文档中心](https://docs.pingcap.com/zh/tidb/dev/basic-features) will be updated in another PR that is particularly for updating the status of all v8.3 features on that page. So it is not updated in this PR to avoid conflict.

image

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -270,4 +270,4 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 
 [^5]: 从 [TiDB v7.0.0](/releases/release-7.0.0.md) 开始新增的参数 `FIELDS DEFINED NULL BY` 以及新增支持从 S3 和 GCS 导入数据，均为实验特性。从 [TiDB v7.6.0](/releases/release-7.6.0.md) 开始 `LOAD DATA` 的事务行为与 MySQL 的事务行为一致，包括事务内的 `LOAD DATA` 语句本身不再自动提交当前事务，也不会开启新事务，并且事务内的 `LOAD DATA` 语句可以被显式提交或者回滚。此外，`LOAD DATA` 语句会受 TiDB 事务模式设置（乐观/悲观）影响。
 
-[^6]: 从 v7.5.0 开始，不再提供 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) 数据同步功能的技术支持，强烈建议使用 [TiCDC](/ticdc/ticdc-overview.md) 实现高效稳定的数据同步。尽管 TiDB Binlog 在 v7.5.0 仍支持 Point-in-Time Recovery (PITR) 场景，但是该组件在未来 LTS 版本中将被完全废弃，推荐使用 [PITR](/br/br-pitr-guide.md) 替代。
+[^6]: 从 v8.3.0 开始，[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) 被废弃，并将在未来版本中移除。如需进行增量数据同步，请使用 [TiCDC](/ticdc/ticdc-overview.md)。如有 Point-in-Time Recovery (PITR) 场景的需求，请使用 [PITR](/br/br-pitr-guide.md)。
```
Comment (lilin90) [inline basic-features.md:273]:
```suggestion
[^6]: 从 v8.3.0 开始，[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) 被废弃，并计划在未来版本中移除。如需进行增量数据同步，请使用 [TiCDC](/ticdc/ticdc-overview.md)。如有 Point-in-Time Recovery (PITR) 场景的需求，请使用 [PITR](/br/br-pitr-guide.md)。
```

File: tidb-binlog/tidb-binlog-overview.md [hunk 2]
```diff
@@ -13,11 +13,10 @@ TiDB Binlog 支持以下功能场景：
 - **数据同步**：同步 TiDB 集群数据到其他数据库。
 - **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。
 
-> **注意：**
+> **警告：**
 >
+> - 从 v8.3.0 开始，TiDB Binlog 被废弃，请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。如有 Point-in-Time Recovery (PITR) 场景的需求，请使用 [PITR](/br/br-pitr-guide.md)。
```
Comment (lilin90) [inline tidb-binlog/tidb-binlog-overview.md:18]:
```suggestion
> - 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。如有 Point-in-Time Recovery (PITR) 场景的需求，请使用 [PITR](/br/br-pitr-guide.md)。
```

File: tidb-binlog/deploy-tidb-binlog.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 aliases: ['/docs-cn/dev/tidb-binlog/deploy-tidb-binlog/','/docs-cn/dev/reference/tidb-binlog/deploy/','/docs-cn/dev/how-to/deploy/tidb-binlog/','/docs-cn/dev/reference/tools/tidb-binlog/deploy/']
-summary: TiDB Binlog 集群部署需要满足服务器硬件配置要求。推荐使用 TiUP 部署 TiDB Binlog，也可以使用 Binary 部署。部署过程中需要注意配置参数和启动命令。在运行 TiDB 时，需要保证至少一个 Pump 正常运行。Drainer 不支持对 ignore schemas 的 table 进行 rename DDL 操作。
+summary: TiDB Binlog 集群部署需要满足服务器硬件配置要求。推荐使用 TiUP 部署 TiDB Binlog，也可以使用 Binary 部署。部署过程中需要注意配置参数和启动命令。在运行 TiDB 时，需要保证至少一个 Pump 正常运行。Drainer 不支持对 ignore schemas 的 table 进行 rename DDL 操作。从 v8.3.0 开始，TiDB Binlog 被废弃，请使用 TiCDC 作为增量数据同步的替代方案。
 ---
 
 # TiDB Binlog 集群部署
 
+> **警告：**
+>
+> 从 v8.3.0 开始，TiDB Binlog 被废弃，请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。
```
Comment (lilin90) [inline tidb-binlog/deploy-tidb-binlog.md:11]:
```suggestion
> 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。
```

File: tidb-binlog-deployment-topology.md [hunk 4]
```diff
@@ -6,7 +6,11 @@ aliases: ['/docs-cn/dev/tidb-binlog-deployment-topology/']
 
 # TiDB Binlog 部署拓扑
 
-本文介绍在部署最小拓扑集群的基础上，同时部署 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。TiDB Binlog 是目前广泛使用的增量同步组件，可提供准实时备份和同步功能。
+本文介绍在部署最小拓扑集群的基础上，同时部署 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。TiDB Binlog 可提供准实时备份和同步功能。
+
+> **警告：**
+>
+> 从 v8.3.0 开始，TiDB Binlog 被废弃，请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。
```
Comment (lilin90) [inline tidb-binlog-deployment-topology.md:13]:
```suggestion
> 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。
```

File: tidb-binlog/bidirectional-replication-between-tidb-clusters.md [hunk 5]
```diff
@@ -1,17 +1,16 @@
...
 
 # 集群间双向同步
 
 > **警告：**
 >
 > - 目前双向同步属于实验特性，尚未经过完备的测试，不建议在生产环境中使用该功能。
-> - TiDB Binlog 与 TiDB v5.0 开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](/tidb-binlog/tidb-binlog-overview.md#注意事项)。
-> - 从 TiDB v7.5.0 开始，TiDB Binlog 组件的数据同步功能不再提供技术支持，强烈建议使用 [TiCDC](/ticdc/ticdc-overview.md) 作为数据同步的替代方案。
-> - 尽管 TiDB v7.5.0 仍支持 TiDB Binlog 组件的实时备份和恢复，但该组件在未来版本中将被完全废弃，推荐使用 [PITR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
+> - 从 v8.3.0 开始，TiDB Binlog 被废弃，请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。如有 Point-in-Time Recovery (PITR) 场景的需求，请使用 [PITR](/br/br-pitr-guide.md)。
```
Comment (lilin90) [inline tidb-binlog/bidirectional-replication-between-tidb-clusters.md:12]:
```suggestion
> - 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。如有 Point-in-Time Recovery (PITR) 场景的需求，请使用 [PITR](/br/br-pitr-guide.md)。
```

File: tidb-binlog/upgrade-tidb-binlog.md [hunk 6]
```diff
@@ -1,18 +1,17 @@
...
 
 如未特别指明，文中出现的 TiDB Binlog 均指最新的 [Cluster](/tidb-binlog/tidb-binlog-overview.md) 版本。
 
-> **注意：**
+> **警告：**
 >
-> - TiDB Binlog 与 TiDB v5.0 开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](/tidb-binlog/tidb-binlog-overview.md#注意事项)。
-> - 从 TiDB v7.5.0 开始，TiDB Binlog 组件的数据同步功能不再提供技术支持，强烈建议使用 [TiCDC](/ticdc/ticdc-overview.md) 作为数据同步的替代方案。
-> - 尽管 TiDB v7.5.0 仍支持 TiDB Binlog 组件的实时备份和恢复，但该组件在未来版本中将被完全废弃，推荐使用 [PITR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
+> - 从 v8.3.0 开始，TiDB Binlog 被废弃，请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。如有 Point-in-Time Recovery (PITR) 场景的需求，请使用 [PITR](/br/br-pitr-guide.md)。
```
Comment (lilin90) [inline tidb-binlog/upgrade-tidb-binlog.md:13]:
```suggestion
> - 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。如有 Point-in-Time Recovery (PITR) 场景的需求，请使用 [PITR](/br/br-pitr-guide.md)。
```

File: ecosystem-tool-user-guide.md [hunk 7]
```diff
@@ -125,6 +125,10 @@ TiDB 提供了 TiUP 和 TiDB Operator 部署运维工具，满足你在不同系
 
 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) 是收集 TiDB 的增量 binlog 数据，并提供准实时同步和备份的工具。该工具可用于 TiDB 集群间的增量数据同步，如将其中一个 TiDB 集群作为另一个 TiDB 集群的从集群。
 
+> **警告：**
+>
+> 从 v8.3.0 开始，TiDB Binlog 被废弃。请使用 [TiCDC](/ticdc/ticdc-overview.md) 进行增量数据同步。
```
Comment (lilin90) [inline ecosystem-tool-user-guide.md:130]:
```suggestion
> 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。。请使用 [TiCDC](/ticdc/ticdc-overview.md) 进行增量数据同步。
```

File: tidb-binlog-deployment-topology.md [hunk 8]
```diff
@@ -6,7 +6,11 @@ aliases: ['/docs-cn/dev/tidb-binlog-deployment-topology/']
 
 # TiDB Binlog 部署拓扑
 
-本文介绍在部署最小拓扑集群的基础上，同时部署 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。TiDB Binlog 是目前广泛使用的增量同步组件，可提供准实时备份和同步功能。
+本文介绍在部署最小拓扑集群的基础上，同时部署 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。TiDB Binlog 可提供准实时备份和同步功能。
+
+> **警告：**
+>
+> 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。
```
Comment (lilin90) [inline tidb-binlog-deployment-topology.md:13]:
```suggestion
> 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 进行增量数据同步。
```

File: tidb-binlog/deploy-tidb-binlog.md [hunk 9]
```diff
@@ -1,11 +1,15 @@
...
 aliases: ['/docs-cn/dev/tidb-binlog/deploy-tidb-binlog/','/docs-cn/dev/reference/tidb-binlog/deploy/','/docs-cn/dev/how-to/deploy/tidb-binlog/','/docs-cn/dev/reference/tools/tidb-binlog/deploy/']
-summary: TiDB Binlog 集群部署需要满足服务器硬件配置要求。推荐使用 TiUP 部署 TiDB Binlog，也可以使用 Binary 部署。部署过程中需要注意配置参数和启动命令。在运行 TiDB 时，需要保证至少一个 Pump 正常运行。Drainer 不支持对 ignore schemas 的 table 进行 rename DDL 操作。
+summary: TiDB Binlog 集群部署需要满足服务器硬件配置要求。推荐使用 TiUP 部署 TiDB Binlog，也可以使用 Binary 部署。部署过程中需要注意配置参数和启动命令。在运行 TiDB 时，需要保证至少一个 Pump 正常运行。Drainer 不支持对 ignore schemas 的 table 进行 rename DDL 操作。从 v8.3.0 开始，TiDB Binlog 被废弃，请使用 TiCDC 作为增量数据同步的替代方案。
 ---
 
 # TiDB Binlog 集群部署
 
+> **警告：**
+>
+> 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 作为增量数据同步的替代方案。
```
Comment (lilin90) [inline tidb-binlog/deploy-tidb-binlog.md:11]:
```suggestion
> 从 v8.3.0 开始，TiDB Binlog 被废弃，并计划在未来版本中移除。请使用 [TiCDC](/ticdc/ticdc-overview.md) 进行增量数据同步。
```

**Comment by lilin90** [comment]
@Benjamin2037 @hfxsd PTAL

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]

**Pull request merged**